cdap-docs/admin-manual/source/installation/mapr/installation.rst

```diff
@@ -0,0 +1,34 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. |hadoop-distribution| replace:: MapR
+
+.. include:: /../target/_includes/mapr-installation.rst
+  :end-before: .. _mapr-install-java-runtime:
+
+As described in these Software Prerequisites, a configured Hadoop and HBase (plus an
+optional Hive client) needs to be configured on the node(s) where CDAP will run.
+
+If colocating CDAP on cluster hosts with actual services, such as the MapR CLDB, Yarn
+ResourceManager, or HBase Master, then the client configurations will already be in place.
+Note that Gateways are redundant if colocating CDAP on cluster hosts with actual services,
```

Comment:
Removed. That section was from the [existing MapR documentation](http://docs.cask.co/cdap/3.2.1/en/integrations/mapr/index.html).